### PR TITLE
F6 add functionality

### PR DESCRIPTION
**Pull request by sushmita12345**
Added Cart ContextProvider 
Added Wishlist ContextProvider 
Added basic functionality on it

### CODE REVIEW

File: src/Reducer/cartReducer.jsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+            return { ...state, cart: [...state.cart, { ...action.payload, qty: 1 }] 
+        };
+      
+        case "REMOVE_FROM_CART":
+        return {
+            ...state,
+            cart: [...state.cart.filter((item) => item.id !== action.payload.id)]
+        };
+
+        case "MOVE_TO_CART":
```
Comment (deekshithmd) [inline src/Reducer/cartReducer.jsx:14]: suggestion : you can eleminate this case by using addtocart case for move to cart functionality

File: src/Reducer/filterReducer.jsx [hunk 2]
```diff
@@ -104,28 +106,27 @@ export function filterReducer(state, action) {
...
+      //     return {
+      //       ...state, cart: [...state.cart, {...action.payload, qty: 1}]
+      //     }
+
+        // case "ADDED_TO_CART":
+        //   return {
+        //     ...state,
+        //     cart: [
+        //       ...state.cart.filter((item) => item.id !== action.payload.id)
+        //     ]
```
Comment (deekshithmd) [inline src/Reducer/filterReducer.jsx]: Remove these commented code if everything working fine

**Review by deekshithmd — APPROVED** [review]

**Comment by muskanbagrecha** [comment]
You can modify the reducer code based on the response we receive from mockbee api. This way you don't need to manually increase the qty as it is happening in the backend.

**Review by muskanbagrecha — APPROVED** [review]
Make the changes as suggested, open a new PR for the cart functionality with mockbee API, will review again.

**Comment by sushmita12345** [comment]
Ok thank you, I'll do that.

**Pull request merged**